include/package-ipkg.mk

```diff
@@ -210,7 +210,7 @@ $(_endef)
 	$(call Package/$(1)/install-overlay,$$(IDIR_$(1))/rootfs-overlay)
 	-find $$(IDIR_$(1)) -name 'CVS' -o -name '.svn' -o -name '.#*' -o -name '*~'| $(XARGS) rm -rf
 	$(Q)( \
-		find $$(IDIR_$(1)) -name lib\*.so\* -or -name \*.ko | awk -F/ '{ print $$$$NF }'; \
+		find $$(IDIR_$(1)) -name lib\*.so\* -o -name \*.ko | awk -F/ '{ print $$$$NF }'; \
```

Comment:
Support for `-or` appears pretty common.  I don't disagree with this, but I'm dubious of the value.